homeassistant/components/sma/entities.py

```diff
@@ -0,0 +1,776 @@
+        device_class=SensorDeviceClass.BATTERY,
+        entity_registry_enabled_default=False,
+    ),
+    "battery_voltage_a": SensorEntityDescription(
+        key="battery_voltage_a",
+        name="Battery Voltage A",
+        native_unit_of_measurement=ELECTRIC_POTENTIAL_VOLT,
+        state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.VOLTAGE,
+        entity_registry_enabled_default=False,
+    ),
+    "battery_voltage_b": SensorEntityDescription(
+        key="battery_voltage_b",
+        name="Battery Voltage B",
+        native_unit_of_measurement=ELECTRIC_POTENTIAL_VOLT,
+        state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.VOLTAGE,
+        entity_registry_enabled_default=False,
+    ),
+    "battery_voltage_c": SensorEntityDescription(
+        key="battery_voltage_c",
+        name="Battery Voltage C",
+        native_unit_of_measurement=ELECTRIC_POTENTIAL_VOLT,
+        state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.VOLTAGE,
+        entity_registry_enabled_default=False,
+    ),
+    "battery_current_a": SensorEntityDescription(
+        key="battery_current_a",
+        name="Battery Current A",
+        native_unit_of_measurement=ELECTRIC_CURRENT_AMPERE,
+        state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.CURRENT,
+    ),
+    "battery_current_b": SensorEntityDescription(
+        key="battery_current_b",
+        name="Battery Current B",
+        native_unit_of_measurement=ELECTRIC_CURRENT_AMPERE,
+        state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.CURRENT,
+    ),
+    "battery_current_c": SensorEntityDescription(
+        key="battery_current_c",
+        name="Battery Current C",
+        native_unit_of_measurement=ELECTRIC_CURRENT_AMPERE,
+        state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.CURRENT,
+    ),
+    "battery_temp_a": SensorEntityDescription(
+        key="battery_temp_a",
+        name="Battery Temp A",
+        native_unit_of_measurement=TEMP_CELSIUS,
+        state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.TEMPERATURE,
+    ),
+    "battery_temp_b": SensorEntityDescription(
+        key="battery_temp_b",
+        name="Battery Temp B",
+        native_unit_of_measurement=TEMP_CELSIUS,
+        state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.TEMPERATURE,
+    ),
+    "battery_temp_c": SensorEntityDescription(
+        key="battery_temp_c",
+        name="Battery Temp C",
+        native_unit_of_measurement=TEMP_CELSIUS,
+        state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.TEMPERATURE,
+    ),
+    "battery_status_operating_mode": SensorEntityDescription(
+        key="battery_status_operating_mode",
+        name="Battery Status Operating Mode",
```

Comment:
Can this be an enum sensor?

https://developers.home-assistant.io/docs/core/entity/sensor#available-device-classes

Reply:
Let's work this out in a follow up.